src/components/kepler-gl.js

```diff
@@ -300,6 +296,7 @@ function KeplerGlFactory(
         mapboxApiUrl,
         mapState,
         uiState,
+        visState,
```

Comment:
why is the entire visState need to be passed here? The other mapFields items come from visState. Can we pick out exactly what it is needed in visState to pass along?

Reply:
we can get rid of it completely. layers is already part of mapFields